### PR TITLE
Admin calendar url hints

### LINKED ISSUE #1169: Provide a list of all supported calendar feeds and the valid URL schema in the calendar creation interface

### PR DESCRIPTION
**Pull request by ivankocienski**
Fixes #1169

### CODE REVIEW

File: app/jobs/calendar_importer/parsers/zarts.rb [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 module CalendarImporter::Parsers
   class Zarts < Xml
```
Comment (kimadactyl) [inline app/jobs/calendar_importer/parsers/zarts.rb:6]: Lets use this opportunity to convert this to a proper ticketsolve importer (i.e. adjust whitelist url, rename etc)

File: app/jobs/calendar_importer/parsers/eventbrite.rb [hunk 2]
```diff
@@ -6,6 +6,9 @@
 
 module CalendarImporter::Parsers
   class Eventbrite < Base
+    NAME = 'Event Brite'
```
Comment (kimadactyl) [inline app/jobs/calendar_importer/parsers/eventbrite.rb:9]: Eventbrite one word

File: app/jobs/calendar_importer/parsers/ics.rb [hunk 3]
```diff
@@ -6,6 +6,13 @@
 
 module CalendarImporter::Parsers
   class Ics < Base
+    NAME = 'ICS'
```
Comment (kimadactyl) [inline app/jobs/calendar_importer/parsers/ics.rb:9]: "Generic iCal / .ics"?

**Review by kimadactyl — CHANGES_REQUESTED** [review]
looks great just a few tweaks for you :)

File: app/jobs/calendar_importer/parsers/zarts.rb (hunk 1 above)
Comment (ivankocienski) [reply app/jobs/calendar_importer/parsers/zarts.rb:6]: so rename Zarts -> TicketSolve (change filename), change the `NAME` and the `DOMAIN = %w[ticketsolve.com]` and the pattern should be something like `/^https?:\/\/([^.]*).ticketsolve.com\.*/` ?

Comment (kimadactyl) [reply app/jobs/calendar_importer/parsers/zarts.rb:6]: Yep exactly (no capitalised S tho by the look of it)

File: app/jobs/calendar_importer/parsers/manchester_uni.rb [hunk 4]
```diff
@@ -4,6 +4,9 @@
 
 module CalendarImporter::Parsers
   class ManchesterUni < Xml
+    NAME = 'Manchester University'
```
Comment (kimadactyl) [inline app/jobs/calendar_importer/parsers/manchester_uni.rb:7]: We should prob have this hidden from the list of generic importers as its a bit confusing. Maybe we could have a "show to public" bool for the importer that prevents it showing up? Or do the move to a subfolder thing I suggested before?

Comment (kimadactyl) [reply app/jobs/calendar_importer/parsers/manchester_uni.rb:7]: I think my pref would prob be the latter just to make it ultra clear whats what in the codebase

**Review by kimadactyl — APPROVED** [review]

**Pull request merged**